source/tutorial/service.md

```diff
@@ -366,27 +366,40 @@ To stub these services we simply have to register a stub service that implements
 
 Add the following code after the imports to our acceptance test:
 
-```/tests/acceptance/list-rentals-test.js{+3,+5,+6,+7,+8,+9,+10,-11,+12,+13,+14,+15,+16,+17}
-import { test } from 'qunit';
-import moduleForAcceptance from 'super-rentals/tests/helpers/module-for-acceptance';
+```/tests/acceptance/list-rentals-test.js{+1,+11,+15,+16,+17,+18,+19,+20,-21,+22,+23,+24,+25,+26,+27,+28,+29}
 import Service from '@ember/service';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import {
+  click,
+  currentURL,
+  find,
+  findAll,
+  visit,
+  fillIn,
+  getContext,
+  triggerKeyEvent
+} from '@ember/test-helpers'
 
 let StubMapsService = Service.extend({
   getMapElement() {
     return document.createElement('div');
   }
 });
 
-moduleForAcceptance('Acceptance | list-rentals');
-moduleForAcceptance('Acceptance | list rentals', {
-  beforeEach() {
-    this.application.register('service:maps', StubMapsService);
-  }
+module('Acceptance | list-rentals');
+module('Acceptance | list rentals', function(hooks) {
+  setupApplicationTest(hooks);
+
+  hooks.beforeEach(function() {
+    let { owner } = getContext();
+    owner.register('service:maps', StubMapsService);
```

Comment:
That getContext() is unnecessary because it's the same as `this`. You can just use this.owner.register()

Reply:
cool. didn't realize owner was available and public these days